### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by Mechtilde** [dpmb/dpmb#26]

**Comment by Mechtilde** [comment]
Hello 

do you also describe the Suffixes +dfsg, +ds, +repack and so on.

**Comment by xtaran** [comment]
Good point. Seems to be missing in `konzepte/software-in-paketen-organisieren/benennung-eines-debian-pakets.adoc`.

**Comment by Mechtilde** [comment]
In the meantime I found this short description https://wiki.debian.org/DebianMentorsFaq#What_does_.2BIBw-dfsg.2BIB0_or_.2BIBw-ds.2BIB0_in_the_version_string_mean.3F

**Comment by hofmannedv** [comment]
Thank you! This helps to complete the list. We are on it.
